### DISCUSSION TITLE
dophin upgrade from 2.x to 3.1.4, web loading long time and task cannot run after a few days.

### DISCUSSION THREAD

**Discussion by cumtqiangqiang** [apache/dolphinscheduler#13875] [Q&A]
after upgrade to 3.1.4 from 2.0.6, found web loading cost too much time and cancelled,
![image](https://user-images.githubusercontent.com/14304089/230091616-5de17c3b-8656-4b38-be72-d058a2e4d03e.png)

and after a few day, job can not run, found host is null.

![image](https://user-images.githubusercontent.com/14304089/230091577-fced55c2-0cb7-4716-9608-ea4b6c1cffb3.png)

**Comment by cumtqiangqiang** [comment]
@Tandoy  Could you give me some advice, many thx

**Reply by Tandoy** [reply]
> @Tandoy Could you give me some advice, many thx

Sorry for replying to you so late. I have no experience upgrading dolphins. Maybe you can try submitting an issue.
